src/plots/geo/geo.js

```diff
@@ -709,6 +709,13 @@ function getProjection(geoLayout) {
         return projection.getPath().bounds(object);
     };
 
+    // https://github.com/d3/d3-zoom/blob/master/README.md#zoom_scaleExtent
+    projection.scaleExtent = function() {
+        var minscale = projLayout.minscale * 100;
+        var maxscale = projLayout.maxscale * 100;
+        return [ minscale, maxscale ];
```

Comment:
We should also handle case of `-1` here. For example:
```js
projection.scaleExtent = function() {
        var minscale = projLayout.minscale;
        var maxscale = projLayout.maxscale;
        if(maxscale === -1) maxscale = Infinity;
        return [ 100 * minscale, 100 * maxscale ];
};
```

Reply:
done in 76cb01f0789e151082612744f1d4218fb091234d